RecoParticleFlow/PFClusterProducer/interface/PFHFRecHitCreator.h

```diff
@@ -60,6 +60,9 @@ class PFHFRecHitCreator final :  public  PFRecHitCreatorBase {
 	const HcalDetId& detid = (HcalDetId)erh.detid();
 	auto depth = detid.depth();
 
+	// ATTN: skip dual anode in HF for now (should be fixed in upstream changes)
+	if( depth > 2 ) continue; 
```

Comment:
There will never be HF RecHits with depth >2. The issue is that right now we have the split anode geometry in 2023, but the associated local reconstruction is still in development. Hopefully all of these pieces will come together soon.